warehouse/forklift/legacy.py

```diff
@@ -501,6 +549,10 @@ class MetadataForm(forms.Form):
         description="Classifier",
         validators=[_validate_no_deprecated_classifiers, _validate_classifiers],
     )
+    dynamic = ListField(
+        description="Dynamic",
+        validators=[_validate_dynamic],
+    )
```

Comment:
This needs to be added as a field of the `Release` model as well: https://github.com/pypi/warehouse/blob/main/warehouse/packaging/models.py#L423-L604